code/src/pages/NotFound.js

```diff
@@ -0,0 +1,2 @@
+
+
```

Comment:
Delate component if you're not using it :)

Reply:
Absolutely, thought I was going to use it but nooo. Gone :)